main.go

```diff
@@ -145,10 +146,12 @@ func findArtifact(rootDir, ext string, buildStart time.Time) ([]string, error) {
 	return matches, nil
 }
 
-func checkBuildProducts(apks []string, apps []string, ipas []string, platforms []string, target string) error {
+func checkBuildProducts(apks []string, aabs []string, apps []string, ipas []string, platforms []string, target string) error {
 	// if android in platforms
-	if len(apks) == 0 && sliceutil.IsStringInSlice("android", platforms) {
-		return errors.New("No apk generated")
+	if sliceutil.IsStringInSlice("android", platforms) {
+		if len(apks) == 0 || len(aabs) == 0 {
+			return errors.New("No apk or aab generated")
```

Comment:
errors should start with a lower case letter: https://github.com/golang/go/wiki/CodeReviewComments#error-strings